fe/fe-core/src/main/java/org/apache/doris/load/BrokerFileGroup.java

```diff
@@ -385,6 +385,13 @@ public void write(DataOutput out) throws IOException {
         // src table
         out.writeLong(srcTableId);
         out.writeBoolean(isLoadFromTable);
+        Text.writeString(out, mergeType.name());
```

Comment:
I think we can change this class to GSON DeSer.

Reply:
this method is useless， I have marked it  as  depricated